salt/metalk8s/volumes/prepared/installed.sls

```diff
@@ -17,3 +17,9 @@ Install gdisk:
   {{ pkg_installed('gdisk') }}
     - require:
       - test: Repositories configured
+
+# Needed by the sparse volume cleanup script
+Ensure Python 2 is available:
+  test.fail_without_changes:
```

Comment:
Well, I would've used a `pkg_installed('python2')` or something along those lines here ;-) But sure, works for this particular case I guess.